plot/steer_aggregate_and_plot_observables.py

```diff
@@ -158,29 +337,80 @@ def main():
+                            if len(debug_design_points) > 0:
+                                if run_dictionary[facility][run]['parametrization']['design_point_index'] not in debug_design_points:
+                                    skipped_runs[facility].append(run)
+                                    runs[facility].remove(run)
+                                    continue
+                            if len(debug_calculation_type) > 0:
+                                if run_dictionary[facility][run]['calculation_type'] not in debug_calculation_type:
+                                    skipped_runs[facility].append(run)
+                                    runs[facility].remove(run)
+                                    continue
+                            if len(debug_system) > 0:
+                                if run_dictionary[facility][run]['system'] not in debug_system:
+                                    skipped_runs[facility].append(run)
+                                    runs[facility].remove(run)
+                                    continue
                         else:
                             run_dictionary[facility][run]['system'] = 'pp'
+                            if len(debug_system) > 0:
+                                if run_dictionary[facility][run]['system'] not in debug_system:
+                                    skipped_runs[facility].append(run)
+                                    runs[facility].remove(run)
+                                    continue
+                            if len(debug_sqrts) > 0:
+                                if run_dictionary[facility][run]['sqrt_s'] not in debug_sqrts:
+                                    skipped_runs[facility].append(run)
+                                    runs[facility].remove(run)
+                                    continue
                 else:
-                    if download_runinfo:
-                        print(f'Warning: {run}_info.yaml not found on OSN')
+                    print(f'Warning: {run}_info.yaml not found!')
                     runs[facility].remove(run)
                     missing_runinfo[facility].append(run)
-
-                if download_runinfo:
-                    print()
+                
+                # if download_runinfo:
+                    # print()
 
         # Print what we found
         for facility in facilities:
             print(f'{facility}:')
             print()
-            print('  We found the following runs:')
+            print(f'  We found the following runs (N ={len(runs[facility])}):')
             print()
             print(f'    {list(dict(run_dictionary[facility]).keys())}')
             print()
             print(f'Warning: We did NOT find run_info for the following runs:')
             print(f'    {missing_runinfo[facility]}')
             print()
             print()
+            print(f'Warning: We skipped the following runs with binomial parametrization:')
```

Comment:
This should be conditional

Reply:
change to a general statement that lists all skipped runs